index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="shortcut icon"
+      href="./Images/service_camp.jpeg"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Service Camp</title>
+  </head>
+  <body>
+    <main>
+      <nav class="social">
+        <ul class="social-container">
+          <li class="social-item"><a href=""></a></li>
+          <li class="social-item"
+            ><a href=""><i class="fa-brands fa-facebook-square"></i></a
+          ></li>
+          <li class="social-item"><a href="">logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <div class="menu">
+          <a href="./index.html"
+            ><img src="./Images/nomiddlemen.png" alt=""
+          /></a>
+          <div class="hamburger"></div>
+        </div>
+        <div>
+          <ul class="list-container">
+            <li class="list-item"><a href="./about.html">About</a></li>
+            <li class="list-item"><a href="">See Product</a></li>
+            <li class="list-item"><a href="">Signup As A Talent</a></li>
+            <li class="list-item"><a href="">Sponsor</a></li>
+            <li class="list-item last-child"
+              ><a href="">No Middleman Campaign</a></li
```

Comment:
- [ ] Please make sure that the links are responsive. in screen size smaller than `1051px` the links ext wraps:
<img width="720" alt="Screenshot 2022-04-15 at 21 00 56" src="https://user-images.githubusercontent.com/77038610/163599192-c77739ad-20d6-4934-8f06-cfbfc4204832.png">

You can address this issue by decreasing the text amount, eg `See Products => Products`, `Sign up as a talent => Sign up` , `No Middleman campaign => No Middleman`:

<img width="667" alt="Screenshot 2022-04-15 at 21 04 18" src="https://user-images.githubusercontent.com/77038610/163599572-179ef4ca-75c7-45e6-8777-3ea9a44cb508.png">
And I think it would be better if you used flex here, the space between the links would be auto adjusted 👍🏻

Reply:
Done